pallets/omnipool/src/lib.rs

```diff
@@ -280,6 +280,10 @@ pub mod pallet {
 			asset_out: T::AssetId,
 			amount_in: Balance,
 			amount_out: Balance,
+			asset_fee: Permill,
+			asset_fee_amount: Balance,
+			protocol_fee: Permill,
```

Comment:
why you are emitting both `Balance` and `Permill` in event? isn't the Balance sufficient? that is only value i am interested in

Reply:
i believe it is useful to have the exact fee as well. so when you look at your trade, you see the event and you'd have exact idea what percentage did you pay. 

the amount itself might not be that clear. i think fee percentage might give users what is the fee quicker than amount alone.

Reply:
but it is redundant, if you have an amount you can calculate the percentage right?

Reply:
that's why i prefer to return Balance which is exact, derived Permill doesn't have to be at there is always room for rouding issues

Reply:
removed.